CMakeLists.txt

```diff
@@ -661,11 +661,16 @@ if(NRN_ENABLE_DOCS)
 
   # Target to execute && convert notebooks to html. Note that neuron must be available for python
   # import, so we use NRN_RUN_FROM_BUILD_DIR_ENV. See docs/README.md.
+  add_custom_target(
+    notebooks_nrnivmodl
+    COMMAND ${PROJECT_BINARY_DIR}/bin/nrnivmodl mod
```

Comment:
I believe we could call `nrnivmodl` directly in the jupyter notebook ?

Reply:
Maybe it would be better to keep it outside so that the jupyter script is cleaner?

Reply:
If we don't want the output, I think there is a way to "hide" it

Reply:
I favor it slightly,  OK to keep as is